src/chrome/content/rules/KSL.xml

```diff
@@ -7,7 +7,13 @@
 	<securecookie host="^\.ksl\.com$" name=".+" />
 
 
-	<rule from="^http://((?:img|local|static|www)\.)?ksl\.com/"
+	<rule from="^http://ksl\.com/"
+		to="https://www.ksl.com/" />
+		<test url="http://ksl.com/" />
+	<rule from="^http://((?:img|local|static|www)\.)ksl\.com/"
```

Comment:
Please list all working hosts as targets and use a trivial `<rule from="^http:" to="https:" />` redirect

Reply:
ping @faazshift